test/intl_suggestors/intl_migrator_test.dart

```diff
@@ -1299,5 +1299,132 @@ void main() {
         expect(messages.messageContents(), expectedFileContent);
       });
     });
+
+    group('Ignore', () {
+      test('Ignore statement with ignore comment', () async {
```

Comment:
As in the other test cases below, triple-quoted strings are really useful for this kind of thing.

Also, in Dart, you can use either single or double quoted strings for the outside (and single triple-quotes or double-triple quotes). Normal convention is to use single-quotes, but when the thing contained is Dart code, that may have its own quoted strings, I prefer to use double quotes for the outer, so the inner part looks more like normal Dart code.

Finally, note that you can have raw strings, so if you don't have any interpolations, you can write
   `r"=> Intl.message('blah $stuff'...);"`
and you don't have to escape the interpolations. It also ignores backslashes, so you can more easily have literal backslashes as well.